.github/workflows/backport.yml

```diff
@@ -0,0 +1,28 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    name: Backport
+    steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
```

Comment:
what is the installation_id mean here? and can't we get it in some variable and then use it rather than hardcoding here.

Reply:
This is the copied from OpenSearch. I kept it same for consistent. In future if they decided to refactor, it will be easy to make change .